src/app/containers/CpsAssetMediaPlayer/utils/getAssetUri/index.js

```diff
@@ -0,0 +1,9 @@
+const getAssetUri = assetUri => {
+  if (assetUri.includes('newyddion')) {
+    return assetUri.replace('newyddion', 'cymrufyw');
```

Comment:
Is this definitely a pattern only used for cymrufyw, seems it would better if this config sat in the service config?

Reply:
It is only for `cymrufyw`. I think we should extract out this logic if we suddenly start needing it in more places.